content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -144,8 +146,8 @@ const Stop: FC<{ stop: Stop }> = ({ stop }) => {
 const ExhibitionCaptions: FC<Props> = ({ stops }) => {
   return (
     <ul className="plain-list no-margin no-padding">
-      {stops.map(stop => (
-        <Stop key={stop.number} stop={stop} />
+      {stops.map((stop: Stop, index) => (
```

Comment:
I think type inference means that you don't need the `Stop` type here – since TS knows that `stops` is an array of `Stop`s (because we told it in props), it can figure it out.

Reply:
You're right of course, thank you